doc/cephadm/osd.rst

```diff
@@ -156,17 +156,29 @@ Example::
 Declarative State
 -----------------
 
-Note that the effect of ``ceph orch apply`` is persistent; that is, drives which are added to the system
-or become available (say, by zapping) after the command is complete will be automatically found and added to the cluster.
+The effect of ``ceph orch apply`` is persistent. This means that drives that
+are added to the system after the ``ceph orch apply`` command completes will be
+automatically found and added to the cluster.  It also means that drives that
+become available (by zapping, for example) after the ``ceph orch apply``
+command completes will be automatically found and added to the cluster.
 
-That is, after using::
+We will examine the effects of the following command:
 
-    ceph orch apply osd --all-available-devices
+   .. prompt:: bash #
+
+     ceph orch apply osd --all-available-devices
+
+After running the above command: 
+
+* If you add new disks to the cluster, they will automatically be used to
+  create new OSDs.
+* If you remove an OSD and clean the LVM physical volume, a new OSD will be
+  created automatically.
 
-* If you add new disks to the cluster they will automatically be used to create new OSDs.
-* A new OSD will be created automatically if you remove an OSD and clean the LVM physical volume.
+To disable the automatic creation of OSD on available devices, use the
+``unmanaged`` parameter:
```

Comment:
Does this mean the command will only create fresh osds from available devices *right now* but not for any devices added in the future?

Reply:
@sebastian-philipp

Reply:
@sebastian-philipp  ping

Reply:
hm. We have three ways:

1. default `ceph orch apply` that will make cephadm constantly reconcile and will create OSDs as soon as new drives appear. 
2. `unamanaged: True` will completely disable the creation of OSDs. So if you apply a **new** OSD service with `unmanaged: True`, then **nothing** will happen.
3. `ceph orch daemon add` will create OSDs but will not add an OSD service. (if I'm not mistaken)

Reply:
I've added these three cases to a note in this section.